### PR TITLE
Connect transform dialect C++ strategy to lowering config

### PR DESCRIPTION
**Pull request by nicolasvasilache**
This PR connects the lowering config added by IREE to the first level of tiling via the transform dialect.

### CODE REVIEW

File: compiler/src/iree/compiler/Codegen/Common/TransformDialectJitterPass.cpp
Comment (matthias-springer) [inline compiler/src/iree/compiler/Codegen/Common/TransformDialectJitterPass.cpp:322]: debugging code left overs

**Review by matthias-springer — APPROVED** [review]

**Comment by MaheshRavishankar** [comment]
Skimmed through the code and it looks fine. In future can you please give some time for folks to review this. I dont think its unreasonable to expect patches to wait for a day or so before they land. We have CODEOWNERS setup so that whoever is on CODEOWNERS get a chance to review code touching relevant parts. So at least CODEOWNERS should get a chance to see it before it lands... if you dont hear in 24 hours, then landing after an approval (without waiting for explicit comments from CODEOWNERS) makes sense to me.

**Pull request merged**
